Jingle/SCJingleConverter/composition.h

```diff
@@ -216,11 +218,13 @@ class Composition
         Note()
             : frequencies(0)
             , length(0.f)
+            , tied(false)
         {}
         std::vector<float> frequencies; // Frequency of note in Hz. Could be multiple
             // frequencies in case of chord. Frequencies to be sorted into descending
             // order at end of parse.
         float length; // Number of beats that Note/Chord lasts for
+        bool tied; // Whether or not this note tries to be tied to the next.
```

Comment:
Change description to something like "True if note is tied to next (i.e. single unbroken note)"